### DISCUSSION TITLE
请问最低支持到安卓哪个版本

### DISCUSSION THREAD

**Discussion by GuengPuak** [yaoxieyoulei/mytv-android#131] [Q&A]
请问最低支持到安卓哪个版本，我的设备是安卓4.22版无法解析此APK呢，是因为系统太老了吗？

**Comment by yaoxieyoulei** [comment]
需android5及以上

**Comment by haseewu** [comment]
目前在售的安卓盒子， 还有不少是android 4.4 的， 用户手上很多还是安卓4.1的， 比如华为秘盒M310, 量都比较大

**Comment by zhangmaorena** [comment]
大佬有计划做5.0以下的吗，几个盒子都用不了😭
